libr/bin/format/mach0/mach0.c

```diff
@@ -2050,6 +2050,12 @@ void *MACH0_(mach0_free)(struct MACH0_(obj_t) *mo) {
 	if (!mo) {
 		return NULL;
 	}
+
+	for (size_t i = 0; !mo->symbols[i].last; i++) {
```

Comment:
do not define variables inside the for loop block, this is part of the c99 standard we don't follow